src/teams/teams.controller.ts

```diff
@@ -59,7 +59,6 @@ export class TeamsController {
     @ApiOperation({
         summary:
             "[Roles: Admin] Gets all teams for a voyage given a voyageId (int).",
-        description: "[access]: admin <br>",
```

Comment:
same here why is the access info removed?

Reply:
Actually I was having a lot of conflicts , it might have been changed when I was resolving them . I will revert it back to as it was.

Reply:
ah ok might be the older version